### PR TITLE
Install lodash library

### PR DESCRIPTION
**Pull request by mrtracy**
Includes typescript typings from DefinitelyTyped.

### CODE REVIEW

**Comment by mrtracy** [comment]
Tamir: I looked into using DefinitelyTyped's TSD package before vendoring this, but I had some concerns about it:

1. The creators still recommend [checking in the typings you download](https://github.com/DefinitelyTyped/tsd#notes).  Apparently it's not yet able to manage versions and dependencies as capably as npm.
2. Looking at their current issues, there appear to be some problems mixing TSD-downloaded files and custom .d.ts files in the same directory.

So I'd like to keep vendoring these typings for now.

**Comment by tamird** [comment]
How come you chose lodash and not lazy.js?

**Comment by tamird** [comment]
LGTM modulo:
- commit message should include rationale for lodash
- go generate

**Comment by BramGruneir** [comment]
@tamird Lazy doesn't seem to be under active development anymore.

**Comment by tamird** [comment]
Ugh :(

**Comment by BramGruneir** [comment]
Which version of LoDash is this?  Modern?

File: ui/js/libs/lodash/3.10.0/lodash.min.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * lodash 3.10.0 (Custom Build) lodash.com/license | Underscore.js 1.8.3 underscorejs.org/LICENSE
+ * Build: `lodash modern -o ./lodash.js`
```
Comment (tamird) [inline ui/js/libs/lodash/3.10.0/lodash.min.js]: @BramGruneir

Comment (BramGruneir) [inline ui/js/libs/lodash/3.10.0/lodash.min.js]: Ah, well ok then.

**Comment by mrtracy** [comment]
It looks like a crapload of lazy evaluation was added into lodash in version 3; it looks like most of their chainable methods support it.

**Comment by tamird** [comment]
Neat. Just add that to the commit message and go forth and merge.

**Pull request merged**